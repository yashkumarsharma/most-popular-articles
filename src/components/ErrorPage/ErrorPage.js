const ErrorPage = () => {
  return (
    <div className='flex justify-center items-center h-screen'>
      Something went wrong! Please try again later!
    </div>
  )
}

export default ErrorPage