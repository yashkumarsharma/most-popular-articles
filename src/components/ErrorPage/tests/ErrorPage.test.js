import React from 'react'
import renderer from 'react-test-renderer'

import ErrorPage from '../ErrorPage'


describe('check ErrorPage', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ErrorPage />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
