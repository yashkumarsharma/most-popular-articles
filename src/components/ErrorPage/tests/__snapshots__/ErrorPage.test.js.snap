// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check ErrorPage renders correctly 1`] = `
<div
  className="flex justify-center items-center h-screen"
>
  Something went wrong! Please try again later!
</div>
`;
